services/gateway/apis/http-version3/swagger/definitions/schemas.json

```
@@ -203,6 +203,299 @@
+                                                    "fieldNumber": {
+                                                        "type": "integer",
+                                                        "example": 1
+                                                    }
+                                                }
+                                            },
+                                            "aggregationBits": {
+                                                "type": "object",
+                                                "properties": {
+                                                    "dataType": {
+                                                        "type": "string",
+                                                        "example": "bytes"
+                                                    },
+                                                    "fieldNumber": {
+                                                        "type": "integer",
+                                                        "example": 2
+                                                    }
+                                                } 
+                                            },
+                                            "certificateSignature": {
+                                                "type": "object",
+                                                "properties": {
+                                                    "dataType": {
+                                                        "type": "string",
+                                                        "example": "bytes"
+                                                    },
+                                                    "fieldNumber": {
+                                                        "type": "integer",
+                                                        "example": 3
+                                                    }
+                                                }
+                                            }
+                                        }
+                                    }
+                                }
+                            },
+                            "signature": {
+                                "type": "object",
+                                "properties": {
+                                    "dataType": {
+                                        "type": "string",
+                                        "example": "bytes"
+                                    },
+                                    "fieldNumber": {
+                                        "type": "integer",
+                                        "example": 15
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    },
     "assetSchema": {
```

Please make necessary changes everywhere else.
```suggestion
    "AssetSchema": {
```